### ISSUE TITLE
login fehlschlag rückmeldung

### ISSUE THREAD

**Issue by joshinils** [JDAPweb/WA-project#92]
wenn ich mich anmelde und das nicht geklappt hat (email oder pw falsch), dann sollte da eine Nachricht erscheinen die mir selbiges mitteilt, und nicht einfach nur die loginseite neu geladen

**Comment by philsupertramp** [comment]
:heavy_check_mark:  definitiv, dazu sollte dann entweder `Username/Email falsch` oder `Passwort falsch` ausgegeben werden.

**Comment by joshinils** [comment]
ich würd einfach ne negativ rückmeldung geben, damit man nicht herausfinden kann, ob die email bei uns existiert in der DB, zwar doof wenn man sich anmeldet, aber besser glaube ich

**Comment by philsupertramp** [comment]
Okay, also sowas wie: `Eingabe falsch` schön fett in rot :smile:

**Comment by joshinils** [comment]
genau, vllt auch kurz eine meldung, dass es geklappt hat mit weiterleitung nach ein paar sekunden, wenn das sinnvoll ist
